doc/dox_comments/header_files/asn_public.h

```diff
@@ -21,6 +21,63 @@
 */
 int wc_InitCert(Cert*);
 
+/*!
+     \ingroup ASN
+
+     \brief This function allocates a new cert structure for use during
+     cert operations without the application having to allocate the structure
+     itself. The cert structure is also initialized by this function removing
+     the need to call wc_InitCert(). When the application is finish using the
+     allocated cert structure wc_CertFree() must be called.
+
+     \return On a successful memory allocation a pointer to the allocated cert
```

Comment:
Sorry to nitpick!  I should have mentioned this earlier but we should probably try to be more consistent with pre-existing API docs. If you look at the section for `wolfSSL_CTX_new()` in `ssl.h` it has the following for return: 

```
    \return pointer If successful the call will return a pointer to the
    newly-created WOLFSSL_CTX.
    \return NULL upon failure.
```
You might want to mention that its zeroized as well here. 

Also, in their example, they check for NULL return which is a best practice that we should also show here: 

```
    ctx = wolfSSL_CTX_new(method);
    if (ctx == NULL) {
        // context creation failed
    }
```

Reply:
Makes sense. Not sure what you mean by "You might want to mention that its zeroized as well here." though. The Cert struct is initialized not zeroised. Is that maybe what you are referring too? I will add that the new struct is initialized.

Reply:
Good point!! 
> The cert structure is also initialized by this function removing the need to call wc_InitCert().

I assumed this meant zeroized.  But "initialized" would be better.